template.yaml

```diff
@@ -179,7 +186,7 @@ Resources:
     Properties:
       CodeUri: src/lambda-edge/check-auth/
       Handler: bundle.handler
-      Runtime: nodejs12.x
+      Runtime: nodejs14.x
```

Comment:
Is it needed to have this here as well as in the globals section? If it's superfluous, best remove it.

(applies all Lambda@Edge functions)